### ISSUE TITLE
Crash on version 1.0.7

### ISSUE THREAD

**Issue by hcoona** [spacechase0/CookingSkill#1]
Environment:
- 1.2-CookingSkill-1.0.7
- SpaceCore v1.0.1
- StardewValley 1.2.33
- SMAPI v1.15
- Windows 10

Console logs:
```
[21:40:05 ERROR SMAPI] An error occured in the base update loop: System.InvalidOperationException: The StardewValley.Game1 object doesn't have a private 'thumbstickToMouseModifier' static field.
   在 StardewModdingAPI.Framework.Reflection.Reflector.GetPrivateField[TValue](Type type, String name, Boolean required) 位置 D:\source\_Stardew\SMAPI\src\StardewModdingAPI\Framework\Reflection\Reflector.cs:行号 57
   在 StardewModdingAPI.Framework.ModHelpers.ReflectionHelper.GetPrivateValue[TValue](Type type, String name, Boolean required) 位置 D:\source\_Stardew\SMAPI\src\StardewModdingAPI\Framework\ModHelpers\ReflectionHelper.cs:行号 110
   在 SpaceCore.Utilities.Reflect.getField[T](Type type, String field) 位置 G:\StardewValley\Mods\SpaceCore\Utilities\Reflect.cs:行号 21
   在 SpaceCore.Overrides.NewGame1.UpdateControlInput(Game1 game, GameTime time) 位置 G:\StardewValley\Mods\SpaceCore\Overrides\NewGame1.cs:行号 264
   在 StardewValley.Game1.Update(GameTime gameTime)
   在 StardewModdingAPI.Framework.SGame.Update(GameTime gameTime) 位置 D:\source\_Stardew\SMAPI\src\StardewModdingAPI\Framework\SGame.cs:行号 593
```

**Comment by hcoona** [comment]
This bug is found in SpaceCore v1.0.1
`thumbstickToMouseModifier` is not a field but a Property so should use `callMethod("get_thumbstickToMouseModifier")` instead.

**Comment by spacechase0** [comment]
Fixed with spacechase0/SpaceCore_SDV@018409e.
